Content.Server/Administration/Systems/BwoinkSystem.cs

```diff
@@ -87,14 +98,128 @@ private void OnOverrideChanged(string obj)
+                NotifyAdmins(e.Session, message, statusType);
+            }
+
             if (e.NewStatus != SessionStatus.InGame)
                 return;
 
             RaiseNetworkEvent(new BwoinkDiscordRelayUpdated(!string.IsNullOrWhiteSpace(_webhookUrl)), e.Session);
         }
 
+        private void NotifyAdmins(ICommonSession session, string message, PlayerStatusType statusType)
+        {
+            if (!_activeConversations.ContainsKey(session.UserId))
+            {
+                // If the user is not part of an active conversation, do not notify admins.
+                return;
+            }
+
+            // Get the current timestamp
+            var timestamp = DateTime.Now.ToString("HH:mm:ss");
+            var roundTime = _gameTicker.RoundDuration().ToString("hh\\:mm\\:ss");
+
+            // Determine the icon based on the status type
+            string icon = statusType switch
+            {
+                PlayerStatusType.Connected => ":green_circle:",
+                PlayerStatusType.Disconnected => ":red_circle:",
+                PlayerStatusType.Banned => ":no_entry:",
+                _ => ":question:"
+            };
+
+            // Create the message parameters for Discord
+            var messageParams = new AHelpMessageParams(
+                session.Name,
+                message,
+                true,
+                roundTime,
+                _gameTicker.RunLevel,
+                playedSound: true,
+                icon: icon
+            );
+
+            // Create the message for in-game with username
+            var color = statusType switch
+            {
+                PlayerStatusType.Connected => "green",
+                PlayerStatusType.Disconnected => "yellow",
+                PlayerStatusType.Banned => "orange",
+                _ => "gray"
+            };
+            var inGameMessage = $"[color={color}]{session.Name} {message}[/color]";
+
+            var bwoinkMessage = new BwoinkTextMessage(
+                userId: session.UserId,
+                trueSender: SystemUserId,
+                text: inGameMessage,
+                sentAt: DateTime.Now,
+                playSound: false
+            );
+
+            var admins = GetTargetAdmins();
+            foreach (var admin in admins)
+            {
+                RaiseNetworkEvent(bwoinkMessage, admin);
+            }
+
+            // Enqueue the message for Discord relay
+            if (_webhookUrl != string.Empty)
+            {
+                if (!_messageQueues.ContainsKey(session.UserId))
+                    _messageQueues[session.UserId] = new Queue<string>();
+
+                var escapedText = FormattedMessage.EscapeText(message);
+                messageParams.Message = escapedText;
+
+                var discordMessage = GenerateAHelpMessage(messageParams);
+                _messageQueues[session.UserId].Enqueue(discordMessage);
```

Comment:
Need to test this, but the contains can be replaced by GetOrNew.
```suggestion
                var queue = messageQueues.GetOrNew(session.UserId);

                var escapedText = FormattedMessage.EscapeText(message);
                messageParams.Message = escapedText;

                var discordMessage = GenerateAHelpMessage(messageParams);
                queue.Enqueue(discordMessage);
```

Reply:
Retested it with this change and it looks all good.
![image](https://github.com/space-wizards/space-station-14/assets/47093363/e655816e-0627-450e-888f-3ebf9af3bfbe)
![image](https://github.com/space-wizards/space-station-14/assets/47093363/a90b7815-7281-4e0a-92da-2d93c460ffbe)